pkg/component/observability/monitoring/prometheus/aggregate/prometheusrules.go

```diff
@@ -67,6 +67,20 @@ func CentralPrometheusRules() []*monitoringv1.PrometheusRule {
 								"summary":     "A node is not healthy.",
 							},
 						},
+						{
+							Alert: "TooManyEtcdSnapshotCompactionJobsFailing",
+							Expr:  intstr.FromString(`count(increase(etcddruid_compaction_jobs_total{succeeded="false"}[3h]) >= 1) / count(increase(etcddruid_compaction_jobs_total[3h]) >= 1) > 0.1`),
+							For:   ptr.To(monitoringv1.Duration("10m")),
```

Comment:
I am not sure about the `for` clause of 10 minutes. The alert considers data for the last 3 hours and I think it feels a bit strange that, with such a time window of 3 hours, we are fine alerting if the condition didn’t recover within just 10 minutes. Usually, I’d expect alerts to consider smaller time windows and alert if the conditions keep failing for longer. You mentioned above that such compaction jobs might run up to 3h so I think 3 hours is okay but maybe, in this case, we could discuss removing the `for` clause so that the alert fires as soon as there are >10% of failing compaction jobs.

Reply:
Yes, I can go ahead and remove the `for` clause since it does not make too much sense given the context that you described here.

Reply:
Adressed in [1fac9a2](https://github.com/gardener/gardener/pull/9739/commits/1fac9a28b83c3dc7bffd27cce5617e37e1cea461).